project.mml

```diff
@@ -1411,6 +1411,22 @@ Layer:
     properties:
       cache-features: true
       minzoom: 12
+  - id: stop-positions
+    geometry: point
+    <<: *extents
+    Datasource:
+      <<: *osm2pgsql
+      table: |-
+        (SELECT
+          way, 
+          tags->'public_transport' as public_transport,
+          ref,
+          tags->'local_ref' as local_ref
+        FROM planet_osm_point
+        WHERE (tags->'public_transport' = 'stop_position') AND (tags->'train' = 'yes') AND ((ref IS NOT NULL) OR ((tags->'local_ref') IS NOT NULL))
```

Comment:
This needs to follow the [SQL style guidelines](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md#sql-style-guidelines)

Reply:
Still an issue.

> To check if a tag is in the tags hstore, use tags @> 'foo=>bar', relying on automatic conversion from text to hstore